sdk/python/feast/job_service.py

```diff
@@ -145,6 +167,54 @@ def GetJob(self, request, context):
         return GetJobResponse(job=self._job_to_proto(job))
 
 
+def run_control_loop():
```

Comment:
should this be `run` or `start`?

Reply:
either way is fine for me, I can change it to start